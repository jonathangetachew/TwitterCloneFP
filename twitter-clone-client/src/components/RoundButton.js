import React from 'react';
import { Button } from 'react-native-elements';
export default (props) => <Button {...props} buttonStyle={[{ marginHorizontal: 2, borderRadius: 20 }, props.buttonStyle]} />