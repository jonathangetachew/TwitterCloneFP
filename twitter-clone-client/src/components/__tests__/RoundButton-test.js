import React from 'react';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import renderer from 'react-test-renderer';

import RoundButton from '../RoundButton';


jest.mock('react-native-elements', () => ({
  Button: 'Button'
}));

describe('RoundButton', () => {
  jest.useFakeTimers();

  beforeEach(() => {
    NavigationTestUtils.resetInternalState();
  });

  it(`renders the button`, () => {
    const tree = renderer.create(<RoundButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
