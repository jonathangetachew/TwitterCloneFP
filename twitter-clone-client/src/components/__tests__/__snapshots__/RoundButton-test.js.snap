// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundButton renders the button 1`] = `
<Button
  buttonStyle={
    Array [
      Object {
        "borderRadius": 20,
        "marginHorizontal": 2,
      },
      undefined,
    ]
  }
/>
`;
